traffic_ops/ort/atstccfg/README.md

```diff
@@ -0,0 +1,43 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+# atstccfg
+atstccfg is a tool for generating configuration files server-side on ATC cache servers.
+
+## Usage
```

Comment:
Would it be good to inform users here that they can get all this information with `atstccfg -h`?

Reply:
Oh, I thought that was just a by-product of printing usage information when given an unrecognized option. Why does that exit with a failure?

Reply:
I added it as an option so it gets printed in its own output and exits with a success, plus is now documented.

Reply:
>Why does that exit with a failure?

Because it didn't successfully generate a config file. Linux exit codes aren't _necessarily_ errors, sometimes they're useful just to distinguish operation. I don't feel that strongly about it though.

Reply:
>a by-product of printing usage information when given an unrecognized option.

In the code, it sort of is. But it works to tell people that. 
I think we should make it print the full config help when given no arguments, too. Just haven't gotten around to that yet.